### ISSUE TITLE
Improve workflow for embargoed datasets

### ISSUE THREAD

**Issue by ahamelers** [CDL-Dryad/dryad-product-roadmap#2473]
**Problem description, and links to relevant helpdesk tickets:**

During curation, when a dataset has been approved for publication (but before it is pushed live), an author requests that it not be released until a certain date or milestone (e.g., associated MS accepted). Keeping these datasets in 'Curation' skews the stats and poses a risk that a curator absentmindedly pushes it live, but putting it in PPR may also skew stats (would present as dataset going from PPR straight to Published).

**Describe the solution you'd like:**

Need another designation ("On Hold" but maybe something that sounds better) or a function like embargoing that is set to time-release data by Date X but that can be published immediately sooner if requested. Difference would be that dataset would not appear publicly in any form unlike embargoed datasets where landing page is active but data are not available

**Comment by ryscher** [comment]
This sounds very similar to the "publication blackout" feature that we discontinued around 2020. It was discontinued because we wanted to encourage data to be published and not languish in a hidden state. 

I worry about the suggested condition of "associated MS accepted" -- what happens if the MS is never accepted? Would the dataset not be published? In this case, we would have expended curation effort, but we would never collect payment. If someone wants to hold their dataset until MS acceptance, they should use PPR, so curators don't spend time on the dataset until we are certain it will be published.

**Comment by ahamelers** [comment]
@ryscher Yes, I'd like more information about the stats and how they are skewed, and why "(would present as dataset going from PPR straight to Published)" would be a bad thing.
